hpc_benchmark/hpc_benchmark.py

```diff
@@ -417,6 +417,8 @@ def run_simulation():
 
     time_simulate_presim = nest.kernel_status["time_simulate"]
```

Comment:
```suggestion
```
This line is no longer needed then.

Reply:
Sorry, did not commit directly, found it easier to also check when doing it myself.